index.html

```diff
@@ -11,6 +11,7 @@
   <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
   <script src="script.js" defer></script>
   <script src="formValidation.js" defer></script>
+  <script src="localDataStorage.js"></script>

```

Comment:
- [ ] The local storage is not being set because the script "localDataStorage.js" does not have the dom elements available when it tries to access them.

![image](https://user-images.githubusercontent.com/60618877/233316671-ee8c9ff0-b6fa-4421-9024-54c3bea9cc0d.png)


👉 You can simply fix this issue by adding defer to the `<script src="localDataStorage.js"></script>` in `index.html`

✅  Like this `<script src="localDataStorage.js" defer></script>`  Cheers 👍